SOOP/SOOP_CO2/incoming_handler.sh

```diff
@@ -50,20 +50,31 @@ handle_netcdf_file() {
 
     if is_imos_soop_co2_file $file; then
         tmp_file_with_sig=`trigger_checkers_and_add_signature $file $BACKUP_RECIPIENT $checks`
+    if [ $? -ne 0 ]; then
+            file_error "Error in NetCDF checking"
+            rm -f  $tmp_nc_file_with_sig
+        fi
     elif is_future_reef_map_file $file; then
         checks='cf'
         tmp_file_with_sig=`trigger_checkers_and_add_signature $file $BACKUP_RECIPIENT $checks`
+    if [ $? -ne 0 ]; then
+            file_error "Error in NetCDF checking"
+            rm -f  $tmp_nc_file_with_sig
+        fi
     else
         file_error_and_report_to_uploader $BACKUP_RECIPIENT "Not an underway CO2 file "`basename $file`
     fi
```

Comment:
You could avoid repeating a few lines by putting the trigger_checkers after this statement...
```bash
if is_imos_soop_co2_file $file; then
    checks='cf imos:1.4'
elif is_future_reef_map_file $file; then
    checks='cf'
else
    file_error_and_report_ ...
fi

tmp_file_with_sig=`trigger_checkers_and_add_signature $file $BACKUP_RECIPIENT $checks`
if [ $? -ne 0 ]; then 
    ...
```